kong/route_service_test.go

```diff
@@ -492,7 +492,7 @@ func TestRoutesValidationTraditionalCompatible(T *testing.T) {
 			msgStartWith: "schema violation (paths.2: invalid regex:",
 		},
 	} {
-		T.Run(tC.name, func(t *testing.T) {
```

Comment:
We should use that `t` in assertions below to fail the subtest and not the whole suite